thrust/thrust/system/detail/sequential/assign_value.h

```diff
@@ -25,8 +25,9 @@
 #elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_MSVC)
 #  pragma system_header
 #endif // no system header
-#include <thrust/system/detail/sequential/execution_policy.h>
 #include <thrust/detail/raw_pointer_cast.h>
+#include <thrust/system/detail/generic/memory.h> // FIXME: I do not know why this needs to be included here
```

Comment:
I have absolutely no idea why this is needed, some circular dependency that this brekas somehow....